plugins/aladino/functions/hasMissingReviews.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 Explore.dev Unipessoal Lda. All Rights Reserved.
+// Use of this source code is governed by a license that can be
+// found in the LICENSE file.
+
+package plugins_aladino_functions
+
+import (
+	"log"
+
+	"github.com/reviewpad/reviewpad/v3/lang/aladino"
+	"github.com/reviewpad/reviewpad/v3/utils"
+)
+
+func HasMissingReviews() *aladino.BuiltInFunction {
+	return &aladino.BuiltInFunction{
+		Type: aladino.BuildFunctionType([]aladino.Type{}, aladino.BuildBoolType()),
+		Code: hasMissingReviewsCode,
+	}
+}
+
+func hasMissingReviewsCode(e aladino.Env, _ []aladino.Value) (aladino.Value, error) {
+	log.Printf("Running builtin hasMissingReviewsCode")
+
+	prNum := utils.GetPullRequestNumber(e.GetPullRequest())
+	owner := utils.GetPullRequestBaseOwnerName(e.GetPullRequest())
+	repo := utils.GetPullRequestBaseRepoName(e.GetPullRequest())
+
+	requestedUsers := e.GetPullRequest().RequestedReviewers
+	requestedTeams := e.GetPullRequest().RequestedTeams
+
+	// Whenever a review is request to a team the name of the team will appear on "Teams" field.
+	// However, once a review is done by a user on that team, the name of the user will appear on "Users" field and the team name will disappear from "Teams" field.
+	// So, whenever a team name appears on "Teams" field, it means that the review is not done yet.
+	if len(requestedTeams) > 0 {
+		return aladino.BuildBoolValue(true), nil
+	}
+
+	// Whenever a review is request to a user and the reviews wasn't yet performed, the name of the user will appear on "Users" field.
+	// So, whenever a user name appears on "Users" field, it means that the review is not done yet.
+	if len(requestedUsers) > 0 {
+		return aladino.BuildBoolValue(true), nil
+	}
```

Comment:
I don't see the need for the comments and the split of the conditionals:
```suggestion
	if len(requestedTeams) > 0 || len(requestedUsers) > 0  {
		return aladino.BuildBoolValue(true), nil
	}
```